guardian/actions-riff-raff
# fix: Use commit sha to identify PR number

## What does this change?
Extends the functionality added in #85 by attempting to identify the PR to comment on from the commit sha using https://docs.github.com/en/rest/commits/commits?apiVersion=2022-11-28#list-pull-requests-associated-with-a-commit [^1].

This allows the commenting feature to work if the triggering event was not `pull_request`, for example in the https://github.com/guardian/dotcom-rendering repository.

## How to test
Observe https://github.com/guardian/dotcom-rendering/pull/9909 having a comment 🎉.

[^1]: Inspired by https://github.com/jwalton/gh-find-current-pr/blob/master/src/main.ts
